app/api/login.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+class Login < Grape::API
+  format :json
+
+  desc 'Login Endpoint'
+  namespace :login do
+    desc 'Login with email and password'
+    params do
+      requires :email, type: String, desc: 'email'
+      requires :password, type: String, desc: 'password'
+    end
+    post do
+
```

Comment:
Probably you wanted to add an empty line before the post block and mistakenly you added it after the block.